src/assets/images/index.ts

```diff
@@ -1 +1,4 @@
-// 명명규칙은 xxxxImg로 통일
\ No newline at end of file
+// 명명규칙은 xxxxImg로 통일
+
+export { default as californiaImg } from './california.png';
+export { default as spainImg } from './spain.png';
```

Comment:
svg를 컴포넌트화해서 export하는 거기 때문에 파일명 시작은 대문자로 해주기!
```suggestion
export { default as CaliforniaImg } from './california.png';
export { default as SpainImg } from './spain.png';
```

Reply:
앗 요거 png 파일이라서 소문자로 해줬는데 png 파일도 컴포넌트화 시켜주는건가요?

Reply:
어억 아니요 svg인 줄 알앗어요 냅둬도 됨!!!